docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "2"
+services:
+    catpol:
+      build: .
+      depends_on:
+        - postgres
+        - pybossa
+      ports:
+        - "8000:8000"
+    redis-master:
+        image: 'redis:3.0-alpine'
+    redis-sentinel:
+        image: 'jvstein/redis-sentinel:latest'
+        depends_on:
+            - redis-master
+        links:
+            - redis-master
+    postgres:
+        image: 'postgres:9.6-alpine'
+        environment:
+            - POSTGRES_USER=pybossa
+            - POSTGRES_PASSWORD=bestpassword
```

Comment:
Let's find another way of defining the password. It shouldn't live, in clear, inside the repo.

Reply:
We are going to handle this as part of the deployment using Travis CI build variables.